src/Ekart/Cart/card-summary.js

```diff
@@ -1,15 +1,28 @@
-import { Button, Card, CardContent, Typography } from "@mui/material";
-import React from "react";
-import { emptyCart } from "./service";
-import { createOrder } from "../Orders/service";
-import { useNavigate } from "react-router-dom";
+import {Button, Card, CardContent, Typography} from "@mui/material";
+import React, {useState} from "react";
+import {emptyCart} from "./service";
+import {createOrder} from "../Orders/service";
+import {useNavigate} from "react-router-dom";
+import {useDispatch} from "react-redux";
+import {setCartItems} from "./cartReducer";
+import SnackbarComponent from "../../Common/snackbar";
 
-function CardSummary({ cartItems, setCartItems }) {
```

Comment:
removed `setCartItems` but it is used on line 18
I'm confused

Reply:
We dont need to pass the reducer method around, we can just import it in the card summary component

Reply:
ohh makes sense